### ISSUE TITLE
Help with local.gulp-config.js

### ISSUE THREAD

**Issue by sinyayadynya** [fourkitchens/emulsify#129]
I'm getting confused after the new "child" theme in emulsify:

I have created my "child" theme, but from then, it is not clear where the local.gulp-config.js should be added, in web/themes/contrib/emulsify/ or in web/themes/custom/my-theme/ ?

Then, I'm also lost with the modification that need to be done to the original gulp-config file (https://github.com/fourkitchens/emulsify-gulp/blob/develop/gulp-config.js).

Among my different attempt, I have tried to add it in web/themes/custom/my-theme/ with no modification, but I'm getting `Local gulp not found in ~/Sites/mounteccorp/web/themes/custom/corp` and `Try running: npm install gulp` which do not seems to be the right way to go.

I have also try to add the local.gulp-config.js to web/themes/contrib/emulsify/, and replace `const themeDir = './';` with `var themeDir = '../custom/my-theme/';`, but this also seems to be wrong.

In short, I'm a bit in a mess and some help would be most welcome.

**Comment by evanmwillhite** [comment]
Sorry for the confusion! We're trying to work on the documentation now, because "child" is actually not the right word ("clone" is a better word). If you created this clone [by running the drush command for Emulsify 2.x](https://github.com/fourkitchens/emulsify#in-a-composer-based-drupal-install-recommended), then there is actually no relationship between the two themes. You've just created a renamed clone (Emulsify is not a parent theme or anything like that). So, you would then run `npm install` (or `yarn install`) inside your new theme and do any of your local work there, including running `npm start` (or `yarn start`) inside it and making any necessary changes to `local.gulp-config.js` there as well.

I hope this helps, but please let us know if you still have any issues. Thanks for trying out the new version!

**Comment by sinyayadynya** [comment]
Yes, now I understand the idea, thanks @evanmwillhite. I went a bit too far, believing that the new version included subtheming, as I remembered that you would possibly include this option in future. 

Anyway, I'm glad to try Emulsify v2, and will do my best to complete my current project before it's next version.
